wpilibc/src/main/native/include/ErrorBase.h

```diff
@@ -7,6 +7,8 @@
 
 #pragma once
 
+#include <set>
+
```

Comment:
This is a big dependency to include everywhere.  If we do this it should be static to the implementation file not a static variable visible in the header.  Also, std::set has quite poor memory characteristics, although they would only happen (I think) if a global error is generated?  How often can that happen?